netplan_cli/cli/commands/apply.py

```diff
@@ -332,21 +333,19 @@ def command_apply(self, run_generate=True, sync=False, exit_on_error=True, state
                 utils.nm_bring_interface_up(loopback_connection)
 
     @staticmethod
-    def is_composite_member(composites, phy):
+    def is_composite_member(netdef: NetDefinition) -> bool:
         """
         Is this physical interface a member of a 'composite' virtual
         interface? (bond, bridge)
         """
-        for composite in composites:
-            for _, settings in composite.items():
-                if not type(settings) is dict:
-                    continue
-                members = settings.get('interfaces', [])
-                for iface in members:
-                    if iface == phy:
-                        return True
 
-        return False
+        # Should this be extended to other types of parent interfaces (vlans, vrfs)
+        bond_link = netdef.links.get('bond')
+        bridge_link = netdef.links.get('bridge')
+
+        is_composite = bool(bond_link) or bool(bridge_link)
```

Comment:
question: what about VRFs? Those can also have member devices.

Reply:
in the end this function was removed